.github/workflows/crowdin-upload.yml

```
@@ -0,0 +1,58 @@
+name: Crowdin Upload
+on:
+  workflow_dispatch:
+  schedule:
+    # runs every day at 11:00 AM UTC
+    - cron: "0 11 * * *"
+jobs:
+  i18n-upload:
+    name: Upload
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Checkout Source Files
+        uses: actions/checkout@v2
+
+      - name: Generate Crowdin Config
+        uses: freecodecamp/crowdin-action@main
+        env:
+          CROWDIN_PERSONAL_TOKEN: ${{ secrets.CROWDIN_CAMPERBOT_SERVICE_TOKEN }}
+          CROWDIN_API_URL: "https://freecodecamp.crowdin.com/api/v2/"
+          CROWDIN_PROJECT_ID: ${{ secrets.CROWDIN_PROJECT_ID }}
```

```suggestion
          CROWDIN_PROJECT_ID: ${{ secrets.CROWDIN_PROJECT_ID_RIR }}
```

To match the others?